projects/elife.yaml

```diff
@@ -913,20 +940,31 @@ generic-cdn:
         standalone:
             ec2: false
         end2end:
+            # unique because: 
+            # - 'fastly.gcslogging.path' is static. log data would be commingled.
+            unique: true
             fastly:
                 gcslogging:
                     bucket: end2end-elife-fastly
                     path: generic-cdn--end2end/
                     period: 600
         continuumtest:
+            # unique because: 
+            # - 'fastly.gcslogging.path' is static. log data would be commingled.
+            unique: true
             fastly:
                 gcslogging:
                     bucket: staging-elife-fastly
                     path: generic-cdn--continuumtest/
                     period: 600
                 bigquerylogging:
                     dataset: "staging"
+                    # todo: missing 'table' name. what is the default?
```

Comment:
todo

Reply:
there are no defaults, it's inheriting from project defaults.

comment removed